recipes/ovito/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "ovito" %}
+{% set version = "2.9.0" %}
+{% set sha256 = "f0fc561d0e995002c48bee6dc5f7e95c69dd52615a735a1707664a7a5f0e19c4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/stuko/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - qscintilla2
+    - sphinx
+    - numpy 
```

Comment:
You likely want to move the various python libs to has as well, but i don't view that as blocking